### PR TITLE
chore: Bump kfp-pipeline-spec to 0.1.3.1

### PR DESCRIPTION
**Pull request by numerology**
**Description of your changes:**
Sorry 0.1.3 was a bad build.

**Checklist:**
- [ ] The title for your pull request (PR) should follow our title convention. [Learn more about the pull request title convention used in this repository](https://github.com/kubeflow/pipelines/blob/master/CONTRIBUTING.md#pull-request-title-convention). 

- [ ] Do you want this pull request (PR) cherry-picked into the current release branch?

    [Learn more about cherry-picking updates into the release branch](https://github.com/kubeflow/pipelines/blob/master/RELEASE.md#cherry-picking-pull-requests-to-release-branch).

### CODE REVIEW

File: sdk/python/setup.py [hunk 1]
```diff
@@ -38,7 +38,7 @@
     'Deprecated',
     'strip-hints',
     'docstring-parser>=0.7.3',
-    'kfp-pipeline-spec>=0.1.0, <0.2.0',
+    'kfp-pipeline-spec>=0.1.0, <0.2.0, !=0.1.3',
```
Comment (chensun) [inline sdk/python/setup.py:41]: This would require a release of `kfp`. How about we just delete kfp-pipeline-spec 0.1.3 from pypi?

Comment (numerology) [reply sdk/python/setup.py:41]: Already deleted.

That said I think we can still leave `!=0.1.3` here just to be on the safer side. WDYT?

Comment (chensun) [reply sdk/python/setup.py:41]: Not really needed, once deleted from PyPi, the version becomes non-discoverable (there might be a few minutes delay).
That said, it's fine to leave it as-is. We can clean it up the next time we touch this file.

/lgtm

Comment (numerology) [reply sdk/python/setup.py]: Make sense. I removed the `!=0.1.3`

**Pull request merged**